app/src/modules/settings/routes/data-model/field-detail/store/alterations/standard.ts

```diff
@@ -26,3 +28,11 @@ export function setSpecialForType(updates: StateUpdates) {
 			set(updates, 'field.meta.special', null);
 	}
 }
+
+function updateInterface(updates: StateUpdates, fn: HelperFunctions) {
+	const interfayce = getInterface(fn.getCurrent('field.meta.interface'));
```

Comment:
typo. we'll have to name it something different too as `interface` is a reserved word in JS

Reply:
Renamed it to `interface_`